### PR TITLE
Minor changes to step-74

### PR DESCRIPTION
**Pull request by peterrum**

### CODE REVIEW

**Comment by peterrum** [comment]
> Actually, never mind -- I'll just include your changes into my follow-up PR.

Too late ;)

**Review by bangerth — APPROVED** [review]

**Comment by bangerth** [comment]
Follow-up to #10932

File: examples/step-74/step-74.cc [hunk 1]
```diff
@@ -202,7 +202,7 @@ namespace Step74
                          const double       cell_extend_left,
                          const double       cell_extend_right)
   {
-    const double degree = std::max(1., static_cast<double>(fe_degree));
+    const double degree = std::max<double>(1, fe_degree);
```
Comment (tjhei) [inline examples/step-74/step-74.cc:205]: I am not a big fan of this change: It makes it less obvious that a conversion int->double is happening and spells out "double" twice.

Comment (bangerth) [reply examples/step-74/step-74.cc:205]: I changed that further in #11467. Let's not worry about it here.

**Comment by peterrum** [comment]
To me it looks like that #11467 integrated most of my comments. So I will close this PR.

**Pull request closed without merging**